.changeset/olive-roses-mix.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Probably we should add an example of each type

Reply:
fixed